PlayCover/View/AppSettingsView.swift

```diff
@@ -106,7 +106,7 @@ struct AppSettingsView: View {
                 Button("settings.reset") {
                     resetCompletedAlert.toggle()
                     settings.reset()
-                }.buttonStyle(GrowingButton()).padding()
+                }.buttonStyle(.borderedProminent).tint(.accentColor).controlSize(.large).padding()
```

Comment:
This button can't have accent color.

Reply:
Indeed my fault for carelessness when reverting changes to this file.